wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -113,10 +115,25 @@ class _PurchaseItemContentState extends State<PurchaseItemContent> {
           color: Colors.grey.shade200,
           height: double.infinity,
           child: Nft3dWidget(
-            url: viewModel.nft.url,
-            cameraControls: true,
-            backgroundColor: AppColors.kBlack,
-          ),
+              url: viewModel.nft.url,
+              cameraControls: true,
+              backgroundColor: AppColors.kBlack,
+              showLoader: showLoader,
+              onProgress: {
```

Comment:
we can easily expose onProgress callback from nft view instead of exposing low level javascript channel.

Reply:
@ahmadrns Please look in to this

Reply:
Sure